deploy.groovy

```diff
@@ -0,0 +1,74 @@
+//Groovy scripts throw lots of WARNING messages, but it is fine
+//https://issues.apache.org/jira/browse/GROOVY-8339
+
+//Edit this file to define the following Strings
+def SET_DOCKER_REG = "ip:port"
+def SET_DOCKER_W = "/path/to/docker/wrapper/"
+
+def DOCKER_REG = System.getenv("DOCKER_REGISTRY")
+if (!DOCKER_REG) {
+    DOCKER_REG = SET_DOCKER_REG;
+}
+
+//The name of the preface for the docker images
+def DOCKER_IMAGE_NAME = "cnxta/ion-ingest"
+//The stack you want the docker compose file named
+def STACK="ingest-stack"
+
+//The path to the Docker Wrapper
+//IMPORTANT!
+//>>> The base gc-docker script has to be in the same location as this file
+
+// docker_wrapper or set_docker_w if docker_wrapper isn't set
+def DOCKER_W = System.getenv("DOCKER_WRAPPER")
+if (!DOCKER_W) {
+    DOCKER_W = SET_DOCKER_W;
+}
+
+//Tests that variables are correct for the docker compose and docker wrapper
+def checkVars(docker_reg, docker_w) {
+    def dir = System.properties['user.dir']
+    def composeFile = new File(dir,"docker-compose.yml")
+    def wrapperFile = new File(docker_w)
```

Comment:
✏️ It would probably be safer to check that the argument was provided first before trying to make a file of it, or fail if it is not provided.

Reply:
Updated to check for a environment variable first.